frontend/app/src/UserSet.tsx

```diff
@@ -138,11 +131,17 @@ const UserSet = ({ handleClick }: any) => {
             aria-labelledby="demo-row-radio-buttons-group-label"
             name="row-radio-buttons-group"
           >
-            <FormControlLabel value="설정" control={<Radio />} label="설정" />
+            <FormControlLabel
+              value="설정"
+              control={<Radio />}
+              label="설정"
+              onChange={() => handleAuthSet('설정')}
```

Comment:
```suggestion
              onChange={() => setAuthSet(true)}
```
`handleAuthSet`에서 추가적으로 해야 할 일이 있을까요? 없다면 빼도 좋을 것 같습니다.